torch/optim/adam.py

```diff
@@ -557,19 +589,21 @@ def _fused_adam(
 ) -> None:
     if not params:
         return
+    if differentiable:
+        raise RuntimeError("_fused_adam is not differentiable")
```

Comment:
Should this be `Adam` with `fused=True` does not support `differentiable=True` ?

Reply:
ok